badge-maker/lib/badge-renderers.js

```diff
@@ -139,7 +132,7 @@ function renderText({
     return { renderedText: '', width: 0 }
   }
 
-  const textLength = preferredWidthOf(content)
+  const textLength = preferredWidthOf(content, { font: '11px Verdana' })
```

Comment:
Where was the font used for measurement defined previously? Was that internal to anafanafo?

Reply:
Yea, previously it was always 110px Verdana.